### PR TITLE
Improve document

### PR DESCRIPTION
**Pull request by tocean**
**Description**
Improve document by:
Add using docker option in installation page
Tell users the time of compiling MSCCL.

### CODE REVIEW

File: docs/getting-started/installation.mdx [hunk 1]
```diff
@@ -18,14 +18,29 @@ Here're the system requirements for MS-AMP.
...
 
-We strongly recommend using [PyTorch NGC Container](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/pytorch). For example, to start PyTorch 2.1 container, run the following command:
+You can try MS-AMP in two ways: Using Docker or installing from source:  
+
+* Using Docker is a convenient way to get started with MS-AMP. You can use the pre-built Docker image to quickly set up an environment for running MS-AMP.  
+* On the other hand, installing from source gives you more control over the installation process and allows you to customize the installation to your needs.
+
+## Use Docker
+
+You can try the lates MS-AMP Docker container with the following commands:
```
Comment (guoshzhao) [inline docs/getting-started/installation.mdx:28]: latest

File: docs/getting-started/installation.mdx [hunk 2]
```diff
@@ -34,7 +49,7 @@ cd MS-AMP
 git submodule update --init --recursive
 ```
 
-If you want to train model with multiple GPU, you need to install MSCCL to support FP8.
+If you want to train model with multiple GPU, you need to install MSCCL to support FP8. Please note that the compilation of MSCCL may take ~40 minites on A100 nodes and ~7 minutes on H100 node.
```
Comment (guoshzhao) [inline docs/getting-started/installation.mdx:52]: minutes

Comment (guoshzhao) [inline docs/getting-started/installation.mdx:52]: The time is mainly depended on CPU? So the time for GPU is inaccurate.

**Review by guoshzhao — APPROVED** [review]

File: docs/getting-started/installation.mdx (hunk 1 above)
Comment (tocean) [reply docs/getting-started/installation.mdx]: Good point. Fixed.

File: docs/getting-started/installation.mdx (hunk 2 above)
Comment (tocean) [reply docs/getting-started/installation.mdx]: Fixed.

**Review by yzygitzh — APPROVED** [review]

**Pull request merged**